app/models/test_track/identity_session_discriminator.rb

```diff
@@ -13,6 +13,10 @@ def participate_in_online_session?
     authenticated_resource_matches_identity?
   end
 
+  def participate_in_unauthenticated_session?
+    web_context? && !controller.respond_to?(authenticated_resource_method_name, true)
```

Comment:
Why would we care about whether it's an authenticated controller or not actually? It doesn't seem like if it's an authenticated controller we should go ahead and create a new visitor and throw it at the server?